examples/tutorials/hybrid_demucs_tutorial.py

```diff
@@ -0,0 +1,352 @@
+# the SDR score. SDR is the signal-to-distortion
+# ratio, essentially a representation to the “quality” of an audio track.
+# 
+
+N_FFT = 4096
+N_HOP = 4
+stft = torchaudio.transforms.Spectrogram(
+    n_fft=N_FFT,
+    hop_length=N_HOP,
+    power=None,
+)
+
+
+######################################################################
+# 5.2 Audio Segmenting and Processing
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# 
+# Below is the processing steps and segmenting 5 seconds of the tracks in
+# order to feed into the spectrogram and to caclulate the respective SDR
+# scores.
+#
+
+def output_results(original_source: torch.Tensor, predicted_source: torch.Tensor, source: str):
+    print("SDR score is:",
+          separation.bss_eval_sources(
+              original_source.detach().numpy(),
+              predicted_source.detach().numpy())[0].mean())
+    plot_spectrogram(stft(predicted_source)[0], f'Spectrogram {source}')
+    return Audio(predicted_source, rate=sample_rate)
+
+
+segment_start = 150
+segment_end = 155
+
+frame_start = segment_start * sample_rate
+frame_end = segment_end * sample_rate
+
+drums_original = download_asset("tutorial-assets/hdemucs_drums_segment.wav")
+bass_original = download_asset("tutorial-assets/hdemucs_bass_segment.wav")
+other_original = download_asset("tutorial-assets/hdemucs_other_segment.wav")
+vocals_original = download_asset("tutorial-assets/hdemucs_vocals_segment.wav")
+
+mix_spec = mixture[:, frame_start: frame_end]
+
+drums_spec = audios["drums"][:, frame_start: frame_end]
+drums, sample_rate = torchaudio.load(drums_original)
+
+bass_spec = audios["bass"][:, frame_start: frame_end]
+bass, sample_rate = torchaudio.load(bass_original)
+
+other_spec = audios["other"][:, frame_start: frame_end]
+other, sample_rate = torchaudio.load(other_original)
+
+vocals_spec = audios["vocals"][:, frame_start: frame_end]
+vocals, sample_rate = torchaudio.load(vocals_original)
+
+######################################################################
+# 5.3 Spectrograms and Audio
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^
+# 
+# In the next 5 cells, you can see the spectrograms with the respective
+# audios. The audios can be clearly visualized using the spectrogram.
+# 
+# The mixture clip comes from the original track, and the remaining
+# tracks are the model output
+# 
+
+# Mixture Clip
+plot_spectrogram(stft(mix_spec)[0], "Spectrogram Mixture")
+Audio(mix_spec, rate=sample_rate)
+
+######################################################################
+# Drums SDR, Spectrogram, and Audio
+# 
+
+# Drums Clip
+output_results(drums, drums_spec, "drums")
+
+######################################################################
+# Bass SDR, Spectrogram, and Audio
+# 
+
+# Bass Clip
+output_results(bass, bass_spec, "bass")
+
+######################################################################
+# Other SDR, Spectrogram, and Audio
+# 
+
+# Other Clip
+output_results(other, other_spec, "other")
+
```

Comment:
Is there a reason why "other" comes before "vocals"? I personally find vocal is most impreesive.

Reply:
Main reason was because this was the order the model had them initially, so I was going in this order. I will change for the presentation side